### PR TITLE
Hi tony pls provide honest thoughts

### PR DESCRIPTION
**Pull request by thinc5**
also idk how to test so yeh i hope some of the logic is correct let me know!

kind recardos,
tom

### CODE REVIEW

**Comment by Assasindie** [comment]
Hey man,
could you please fix the failing checks thank you!
Kind Regards
Assasindie

**Comment by thinc5** [comment]
hi friend figured out how to and i am right now! please let me know if !f feedbrad is back up by posting on your mastodon page!!

File: src/command/modules/user/Unregister.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+                // Check user does currently exist
+                const doc = await DBUserSchema.getUserById(user.id);
+                if (doc) {
+                    // User does exist
+                    const stat = await DBUserSchema.removeUser(user.id);
+                    if (stat) {
+                        return true;
+                    }
+                } else {
+                    state.getMessageHandle().reply("You are not currently regeristered.");
```
Comment (aXises) [inline src/command/modules/user/Unregister.ts]: Typo here

**Comment by Assasindie** [comment]
LETS HOPE THIS WORKS @aXises

**Pull request merged**